pages/utilities/image-resizer.tsx

```diff
@@ -0,0 +1,284 @@
+import { useCallback, useState, useMemo, ChangeEvent } from "react";
+import PageHeader from "@/components/PageHeader";
+import { Card } from "@/components/ds/CardComponent";
+import { Button } from "@/components/ds/ButtonComponent";
+import { Label } from "@/components/ds/LabelComponent";
+import Header from "@/components/Header";
+import { CMDK } from "@/components/CMDK";
+import CallToActionGrid from "@/components/CallToActionGrid";
+import Meta from "@/components/Meta";
+import ImageResizeSEO from "@/components/seo/ImageResizeSEO";
+import {
+  Format,
+  handleResizeImage,
+  processImageFile,
+  updateHeight,
+  updateWidth,
+} from "@/components/utils/resize-image.utils";
+import { Combobox } from "@/components/ds/ComboboxComponent";
+import { Checkbox } from "@/components/ds/CheckboxComponent";
+import { Input } from "@/components/ds/InputComponent";
+import { ImageUploadComponent } from "@/components/ds/ImageUploadComponent";
+import { cn } from "@/lib/utils";
+import { DownloadIcon } from "lucide-react";
+
+const MAX_DIMENSION = 1024 * 4;
```

Comment:
For image resizer, I feel like we should allow users to upload bigger images, at least up to 40MB.
In other cases we can keep 4MB as it was before